FrontMen/fm-hours
# Feature/multilingual support

# Changes
Added translations for previously hardcoded strings and localization for dates

## Related issues

closes https://github.com/FrontMen/fm-hours/issues/138

## Added

nuxt-i18n

## How to test

navigate app - english wording should still be visible
nederlands should show "#required" for all text
dates should update 
navigation should persist the locale chosen and "nl" is added to url
